src/component/feature/FeatureToggleList/FeatureToggleListTable/FeatureLinkCell/FeatureLinkCell.styles.ts

```diff
@@ -26,9 +26,5 @@ export const useStyles = makeStyles()(theme => ({
         textDecoration: 'none',
         fontSize: 'inherit',
         display: 'inline-block',
-        maxWidth: '250px',
-        whiteSpace: 'nowrap',
```

Comment:
What happens here if the text overflows the container? We can't just cut the text for feature toggles because the name is the id of the feature toggle, and just cutting it off without any tooltip or some way to see the full name won't work. We'll need to use the StringTruncator component or something similar in situations where we have to trim the user input because of length.

Reply:
That makes sense, however these changes apply to the description, not the name. This is the current behaviour: 
![image](https://user-images.githubusercontent.com/14320932/168018804-1fd04f29-6c43-44c2-91bf-7faf2efdef90.png)

This means that the description now wraps to a new line if needed, which also means this is a lot more responsive now, since it behaves properly on all screen sizes. This way we are also able to highlight all parts of the description when searching.

We are aligned on this from a UI/UX perspective, and if the descriptions get too long, we can always truncate them later. Does this make sense for now?